test/cases/040_packages/008_format_mount/000_auto/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# SUMMARY: Check that the format and mount packages work
+# LABELS:
+# REPEAT:
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+NAME=test-format
+
+clean_up() {
+	find . -depth -iname "${NAME}*" -not -iname "*.yml" -exec rm -rf {} \;
+}
+
+trap clean_up EXIT
+
+# Test code goes here
+moby build -name ${NAME} -output kernel+initrd test.yml
+RESULT="$(linuxkit run qemu -disk file=${NAME}1.img,size=512M ${NAME})"
```

Comment:
yes. the device name of drives varies from backend to backend e.g `/dev/vda1` on Hyperkit and `/dev/sda1` on qemu. As some of theses tests require a device name it made sense to me to pin the backend to qemu as it's available on all platforms.